Final_Project/FrameHandler.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <windows.h>
+#include <stdio.h>
+#include <string.h>
+#include "Main.h"
+
+static char nextFrameToReceive = (char)17;
+static char nextFrameToSend = (char)17;
+static bool isClearToSend = true;
+
```

Comment:
Everything looks good! The only thing is that it would be nice to have static values for each ASCII dec character we use. This header file seems to be the best place to put that.